README.md

```diff
@@ -2,12 +2,35 @@
 
 This is the repository hosting the public [DocSearch](https://community.algolia.com/docsearch/) configurations.
 
-## Configuration format
+DocSearch is composed by 3 different projects:
+* The front-end JavaScript library: https://github.com/algolia/docsearch
+* The scraper which browses & indexes web pages: https://github.com/algolia/docsearch-scraper
+* The configurations for the scraper: https://github.com/algolia/docsearch-configs
+
+If you want to run your own DocSearch instance on those configuration files, please follow first the [scraper setup guidelines](https://github.com/algolia/docsearch-scraper).
+
+## Introduction
+
+The DocSearch scraper will use a configuration file specifying:
+ - the Algolia index name to push the resulting crawl to,
```

Comment:
- the Algolia index name that will store the records resulting from the crawling